docs/translations/api-docs/link/link.json

```diff
@@ -12,6 +12,111 @@
   },
   "classDescriptions": {
     "root": { "description": "Styles applied to the root element." },
+    "body2": {
```

Comment:
A quick note, this is a regression. None of these new classes exists in the Link implementation.

Reply:
Please read the description of my PRs.